packages/blade/src/components/Typography/BaseText/BaseText.tsx

```diff
@@ -2,28 +2,17 @@ import type { ReactElement } from 'react';
 import getIn from '../../../utils/getIn';
 import { useTheme } from '../../BladeProvider';
 import makeTypographySize from '../../../utils/makeTypographySize';
-import type { ColorContrast } from '../../../tokens/theme/theme';
 import type { Theme } from '../../BladeProvider';
+import type { DotNotationColorStringToken } from '../../../@types/helpers';
```

Comment:
suggestion: shall we add types to `_helpers/types` like we have `_helpers/testing`?

Reply:
Works 👍